R/S3.R

```diff
@@ -456,6 +457,8 @@ pander.anova <- function(x, caption = attr(x, 'caption'), ...) {
     if (is.null(caption))
         caption <- get.caption()
 
+    if(add.significance.stars && !is.null(x$`Pr(>F)`))
+      x$` ` <- format(stars.pval(x$`Pr(>F)`))
```

Comment:
What is `stars.pval`? Wouldn't it be easier to do this instead of creating a new column by something like https://github.com/warnes/pander/blob/master/R/S3.R#L325